resources/themes/cura-light/theme.json

```diff
@@ -433,6 +439,10 @@
         "objects_menu_size": [20, 40],
         "objects_menu_size_collapsed": [20, 17],
         "build_plate_selection_size": [15, 5],
-        "objects_menu_button": [0.3, 2.7]
+        "objects_menu_button": [0.3, 2.7],
+
+        "toolbox_thumbnail_small": [6.0, 6.0],
+        "toolbox_thumbnail_medium": [8.0, 8.0],
+        "toolbox_thumbnail_large": [12.0, 12.0]
```

Comment:
Maybe it's an idea to merge this with the icon size, like the one we use for the top bar. If that is possible in the design constraints, at least...

Reply:
I don't think so because they're not really "icons" per say. They're also way bigger than any 12-20 px icon in the UI.